src/Foreign/Object.js

```diff
@@ -113,8 +111,8 @@ function toArrayWithKey(f) {
   };
 }
 
-exports.toArrayWithKey = toArrayWithKey;
+export {toArrayWithKey};
```

Comment:
Should we just `export function toArrayWithKey(f) { ... }`?

Reply:
Yeah, I like that better.